### PR TITLE
[fix] fix connect wallet with disableModal flag

### PR DESCRIPTION
**Pull request by applicature-dlitvinov**
### Description
Set inProgress to false if connection error.
Fix for #1032 

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] This PR passes the Circle CI checks

### CODE REVIEW

**Comment by applicature-dlitvinov** [comment]
@Adamj1232 @mahmud-bn @taylorjdawson please review PR.

**Review by YugasVasyl — APPROVED** [review]

File: packages/core/src/index.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ export type {
 } from './types'
 
 export type { EIP1193Provider } from '@web3-onboard/common'
+export { connectWallet$ } from './streams'
```
Comment (Adamj1232) [inline packages/core/src/index.ts:45]: Is this necessary?

**Review by Adamj1232 — CHANGES_REQUESTED** [review]

File: .idea/workspace.xml [hunk 2]
```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (applicature-dlitvinov) [reply .idea/workspace.xml]: done

File: packages/core/src/index.ts (hunk 1 above)
Comment (applicature-dlitvinov) [reply packages/core/src/index.ts]: You are right, It's not necessary. Removed.

Comment (applicature-dlitvinov) [reply packages/core/src/index.ts]: @Adamj1232 fixed all comments. Pls, review again.

**Comment by applicature-dlitvinov** [comment]
@Adamj1232 fixed all comments. Pls, review again.

**Pull request merged**